itwin-platform-access/imodels-access-backend/README.md

```diff
@@ -0,0 +1,7 @@
+# @itwin/imodels-access-backend
+
+Copyright © Bentley Systems, Incorporated. All rights reserved. See [LICENSE.md](../../LICENSE.md) for license terms and full copyright notice.
+
+## About this package
+
+This package contains an implementation of [`BackendHubAccess`](https://github.com/iTwin/itwinjs-core/blob/master/core/backend/src/BackendHubAccess.ts) interface. The implementation uses the [`@itwin/imodels-client-authoring`](../../clients/imodels-client-authoring/README.md) package and adapts it to the defined interface thus enabling the applications built using [iTwin.js](http://www.itwinjs.org) platform to use [iModels API](https://developer.bentley.com/apis/imodels/) for iModel storage.
```

Comment:
We should refer to iTwin.js as library, not a platform. 
Also I would like to avoid referring to iModels API as only storage, it is a bit more. To use iModels API for iModel management.

Reply:
Fixed: https://github.com/iTwin/imodels-clients/blob/8d1ba8536b41d0b6bb1aac2118b9ac456c6a0324/itwin-platform-access/imodels-access-backend/README.md?plain=1#L7
About the storage - I agree with the idea but the word "management" is already used when describing workflows that do not write anything into iModel (named version creation, etc.). So maybe something that includes "repository"?

Reply:
Fixed here: https://github.com/iTwin/imodels-clients/pull/56/commits/aa1efde973fd37163b308977d57b82b99527576e